hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteMergeOnRead.java

```diff
@@ -66,6 +69,7 @@ public void testIndexStateBootstrapWithMultiFilesInOneSlice() throws Exception {
   @Test
   public void testIndexStateBootstrapWithCompactionScheduled() throws Exception {
     // sets up the delta commits as 1 to generate a new compaction plan.
+    conf.set(FlinkOptions.COMPACTION_ASYNC_ENABLED, true);
     conf.setInteger(FlinkOptions.COMPACTION_DELTA_COMMITS, 1);
```

Comment:
The test case is for compaction scheduled but not executed, so we should not add this config option.